src/components/network/network-table.js

```diff
@@ -8,34 +8,49 @@
 /**
  * NetworkTable renders given multi-dimensional data as a table.
  * @param {!Object} data Data object of the view.
+ * @extends {genotet.ViewLoader}
```

Comment:
I just use the signal function to communicate with other parts. So what should I do to communicate with other parts.